client/src/components/MultiSelect.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import CloseIcon from '../assets/CloseIcon';
+
+const InputWrapper = styled.div`
+  display: flex;
+  flex-flow: wrap row;
+  justify-content: center;
+  align-items: center;
+  border: 1.5px solid #707070;
+  border-radius: 5px;
+  padding: 5px 10px;
+  margin: 10px 20px;
+`;
+
+const TagContainer = styled.ul`
+  display: flex;
+  flex-flow: wrap row;
+  justify-content: center;
+  border: none;
+  margin: 0;
+  padding: 0;
+  &:focus {
+    outline: grey;
+  }
+`;
+
+const Option = styled.li`
+  background: #f2d22e;
+  display: flex;
+  color: white;
+  border: none;
+  font-family: Roboto sans-serife;
+  font-size: 1rem;
+  text-align: center;
+  list-style: none;
+  padding: 4px;
+  margin: 2px;
+  cursor: pointer;
+  border-radius: 10px;
+`;
+
+const Input = styled.input`
+  padding: 5px;
+  border: none;
+  cursor: pointer;
+  margin-left: 10px;
+  font-size: 1rem;
+  flex: 1;
+  outline: none;
+  &:placeholder {
+    color: #9eaebc;
+  }
+`;
+
+function MultiSelectInput() {
+  const [tags, setTags] = React.useState([]);
+  const addTags = (event) => {
```

Comment:
`addTags` sounds like the first argument are the tags, but it is the keyUp event.
I would rename it to `handleKeyUp`.
Inside the `if` you add a single tag, which could be another function called `addTag(tag)`.

Reply:
Thank you for the suggestion I also wasn't really happy with the naming. 😅